grails-app/controllers/org/pih/warehouse/order/OrderController.groovy

```diff
@@ -1096,6 +1096,66 @@ class OrderController {
     }
 
     def orderSummary = {
+        Order order = Order.get(params.id)
+        def orderItems = order?.orderItems
+        render(template: "orderSummary", model: [
+            isPurchaseOrder: order.isPurchaseOrder,
+            isPutawayOrder: order.isPutawayOrder,
+            orderItems: orderItems,
+            anySupplierCode: orderItems?.any { it.productSupplier?.supplierCode },
+            anyManufacturerName: orderItems?.any { it.productSupplier?.manufacturerName },
+            anyManufacturerCode: orderItems?.any { it.productSupplier?.manufacturerCode },
```

Comment:
I don't love this "any*" thing. If we need to do it this way, let's just grab "any" product supplier once and use the supplierCode, manufacturerName, manufacturerCode from that supplier.